### PR TITLE
Wasm gemm interface update

### PR DESCRIPTION
**Pull request by abhi-agg**
### Description

This PR improves on the interface introduced in https://github.com/browsermt/marian-dev/pull/49

List of changes:
- Added separate `scale` factor and `zero point` for both A and B in `int8PrepareBias` and `int8MultiplyAndAddBias` interface instead of pre-computing the unquantization value and passing it as a single parameter
- An additional unquantization multiplier parameter in `int8MultiplyAndAddBias`
- Updated the corresponding fallback implementation

Added dependencies: none

### Checklist

- [x] I have tested the code manually
- [ ] I have run regression tests
- [x] I have read and followed CONTRIBUTING.md
- [ ] I have updated CHANGELOG.md

### CODE REVIEW

**Comment by abhi-agg** [comment]
@kpu I made these changes as per our discussion in the last plenary meeting 👍🏾

File: src/tensors/cpu/wasm_intgemm_fallback.cpp [hunk 1]
```diff
@@ -59,18 +59,22 @@ extern "C" void int8PrepareBFromQuantizedTransposedFallback(const int8_t* input_
...
+                                        float zero_point_A,
+                                        float scale_B,
+                                        float zero_point_B,
+                                        float unquant_multiplier,
                                         Index width,
                                         Index cols_B,
                                         const float* input_bias,
                                         float* output) {
   LOG(info, "Calling fallback implementation of \"int8PrepareBias\"");
+  float unquant_factor = (-1) * ((127.0f / scale_A) * (127.0f / scale_B)) / (127.0f);
```
Comment (kpu) [inline src/tensors/cpu/wasm_intgemm_fallback.cpp:72]: I think this should be `unquant_multiplier *= `?

**Comment by kpu** [comment]
I can't think of a use of the `unquant_multiplier` in prepareBias since it's just supposed to make the bias term ready.  You seem you agree with that by not using it.  Suggest removing it as an argument from the prepareBias call.

File: src/tensors/cpu/wasm_intgemm_fallback.cpp (hunk 1 above)
Comment (abhi-agg) [reply src/tensors/cpu/wasm_intgemm_fallback.cpp:71]: You are right. I removed `unquant_multiplier` from prepareBias API altogether as per your recommendation 👍🏾

**Comment by abhi-agg** [comment]
@kpu I removed `unquant_multiplier` from prepareBias API as per your recommendation 👍🏾

**Review by kpu — APPROVED** [review]

**Pull request merged**